test/imports/LuneOS/Service/LunaService.qml

```diff
@@ -108,14 +109,22 @@ QtObject {
         {
             getConnectionManagerStatus_call(args, returnFct, handleError);
         }
-        else if (serviceURI === "palm://com.palm.display/control/getProperty") {
+        else if (serviceURI === "palm://com.palm.display/control/getProperty" || serviceURI === "luna://com.palm.display/control/getProperty") {
             getDisplayProperty_call(args, returnFct, handleError);
         }
         else if(serviceURI ==="luna://com.palm.db/find")
         {
             findDb_call(args, returnFct, handleError);
         }
-
+		else if(serviceURI ==="luna://com.palm.db/search")
+        {
+            findDb_call(args, returnFct, handleError);
```

Comment:
Yes it does, see http://www.openwebosproject.org/docs/developer_guide/tools/db8_examples/#.Vii0MStW_FM